### PR TITLE
"Log System" damn right

### PR DESCRIPTION
**Pull request by ChachyGit**

### CODE REVIEW

File: src/main/java/club/thepufferfishproject/pmlinstaller/ui/frames/FrameStart.java [hunk 1]
```diff
@@ -18,13 +22,25 @@
...
     public void start() {
         System.out.println("Starting FrameStart");
 
         Dimension dim = Toolkit.getDefaultToolkit().getScreenSize();
         JButton start = new JButton("Start");
         JFrame frame;
+
+
+        System.out.println(log.getAbsolutePath());
+        System.out.println("Used space: " + log.getTotalSpace());
```
Comment (ConorTheDev) [inline src/main/java/club/thepufferfishproject/pmlinstaller/ui/frames/FrameStart.java]:
```suggestion
        System.out.println("Used space: " + log.getTotalSpace());
```

What's this needed for?

Comment (ConorTheDev) [reply src/main/java/club/thepufferfishproject/pmlinstaller/ui/frames/FrameStart.java]: +1

**Comment by ConorTheDev** [comment]
Awesome, thanks.

**Comment by ChachyGit** [comment]
ayyy

**Pull request merged**